docs/topics/create-first-yaml-rule.adoc

```diff
@@ -0,0 +1,162 @@
+        </loader-repository>
+    </class-loading>
+</jboss-web>
+----
+
+[id="mta-creating-the-rule_{context}"]
+== Creating the rule
+
+MTA YAML-based rules use the following rule pattern:
+
+[options="nowrap",subs="attributes+"]
+----
+when(condition)
+  perform(action)
+----
+
+.Procedure
+
+. Create a YAML file in the directory you created with the following contents:
++
+[options="nowrap",subs="attributes+"]
+----
+- ruleID: <UNIQUE_RULE_ID> <1>
+  description: <DESCRIPTION> <2>
+  when:
+    <CONDITION(S)> <3>
+  message: <MESSAGE> <4>
+  labels: <LABELS> <5>
+  effort: <EFFORT> <6>
+  links:
+  - <LINKS> <7>
+----
++
+<1> Unique ID for your rule, for instance, `jboss5-web-class-loading`
+<2> Text description that describes the rule
+<3> Complete the `when` block specifying one or more conditions:
+.. Use the `builtin` provider’s XML capability because this rule checks for a match in an XML file.
+.. To match on the `class-loading` element that is a child of `jboss-web`, use the XPath expression `jboss-web/web-loading` as an XML query. In this case, you need just one condition:
++
+[options="nowrap",subs="attributes+"]
+----
+when:
+  builtin.xml:
+    xpath: jboss-web/class-loading
+----
+<4> Helpful message explaining the migration issue. The message is generated in the report when the rule matches. For example:
++
+[options="nowrap",subs="attributes+"]
+----
+message: The class-loading element is no longer valid in the jboss-web.xml file.
+----
+<5> List of string labels for the rule
+<6> Number of expected story points to fix this issue
+<7> One or more hyperlinks pointing to documentation around the migration issues that you find
++
+[options="nowrap",subs="attributes+"]
+----
+links:
+- url: https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/Migration_Guide/index.html#Create_or_Modify_Files_That_Control_Class_Loading_in_JBoss_Enterprise_Application_Platform_6
+  title: Create or Modify Files That Control Class Loading in JBoss EAP 6
+----
++
+The rule is now complete and looks similar to the following:
++
+[options="nowrap",subs="attributes+"]
+----
+- ruleID: jboss5-web-class-loading
+  description: Find class loading element in JBoss XML file.
+  when:
+    builtin.xml:
+      xpath: jboss-web/class-loading
+  message: The class-loading element is no longer valid in the jboss-web.xml file.
+  effort: 3
+  links:
+  - url: https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/Migration_Guide/index.html#Create_or_Modify_Files_That_Control_Class_Loading_in_JBoss_Enterprise_Application_Platform_6
+    title: Create or Modify Files That Control Class Loading in JBoss EAP 6
+----
+
+[id="mta-installing-the-rule_{context}"]
+== Installing the rule
+
+.Procedure
+. Copy the rule file into the rules directory.
+. Point the CLI to the rules directory:
++
+[options="nowrap",subs="attributes+"]
+----
+–rules /home/<USER>/rules/
```

Comment:
@sbeskin-redhat suggesting changing this to 

```
---rules /home/<USER>/rules/rule.yaml
```

Reply:
@pranavgaikwad 
in our Google doc file, you omitted the `rules` directory: home/<USER>/rule.yaml
Which is correct?`

Reply:
@sbeskin-redhat sorry for confusion. Both are correct, but I updated now.